client/db.go

```diff
@@ -140,11 +139,16 @@ type DB struct {
 // Open creates a new database handle to the cockroach cluster specified by
 // addr. The cluster is identified by a URL with the format:
 //
-//   (http|https|rpc|rpcs)://[<user>@]<host>:<port>[?certs=<dir>]
+//   <sender>://[<user>@]<host>:<port>[?certs=<dir>]
 //
-// The rpc and rpcs schemes use a variant of Go's builtin rpc library for
+// The URL scheme (<sender>) specifies which transport to use for talking to
+// the cockroach cluster. Currently allowable values are: http, https, rpc,
+// rpcs. The rpc and rpcs senders use a variant of Go's builtin rpc library for
 // communication with the cluster. This protocol is lower overhead and more
-// efficient than http.
+// efficient than http. The decision between the encrypted (https, rpcs) and
+// unencrypted senders (http, rpc) depends on the settings of the cluster. A
+// given cluster will not support both encrypted and unencrypted traffic
```

Comment:
s/will not support both encrypted and unencrypted traffic simultaneously/supports either encrypted or unencrypted traffic, but not both/